app/frontend/covid/volunteer_groups.js

```diff
@@ -129,15 +129,16 @@ const VolunteerGroups = {
 
 Hope you are all doing well! Thank you for all that you are doing to work on a solution for our world and being part of helpwithcovid.com.
 
-This project needs your help: ${projectName} (http://helpwithcovid.com/projects/${projectId}) by project owner - ${projectOwnerEmail}. I thought you might be able to help. I have included the project owner to this email should have any questions.
+This project needs your help: ${projectName} (http://helpwithcovid.com/projects/${projectId}) by project owner - ${projectOwnerEmail}. I thought you might be able to help. I have included the project owner to this email should you have any questions.
 
-If you are already on an existing project or feel that you are not the right fit after speaking with the project owner. Please login to helpwithcovid.com and remove the setting "pair me with a project" from My Profile page.
+If you are already on an existing project, please login to helpwithcovid.com and remove the setting "pair me with a project" from your profile page (https://helpwithcovid.com/users/edit).
+
+If you feel that you are not the right fit after speaking with the project owner, please go to the project link (http://helpwithcovid.com/projects/${projectId}) and “cancel volunteer offer”. We will work on matching you to another project.
```

Comment:
@caseypugh this is not accurate, they have to go to profile and uncheck pair me with prorjects.

Reply:
@sradu Not sure I follow what to fix exactly. I copied what @prophesy604 wrote in #195. Can one of you edit that and i can make changes?

Reply:
@caseypugh @sradu  on line #134: If you are already on an existing project, please login to helpwithcovid.com and remove the setting "pair me with a project" from your profile page (https://helpwithcovid.com/users/edit).

I used the word "setting" and Radu used the word "uncheck" in the comment above.